kyuubi-server/web-ui/src/layout/components/aside/types.ts

```diff
@@ -94,5 +94,10 @@ export const MENUS = [
     label: 'Contact Us',
     icon: 'PhoneFilled',
     router: '/contact'
+  },
+  {
+    label: 'SQL Lab',
+    icon: 'Cpu',
```

Comment:
kind of an awkward expression... does "CPU"(prefered) or "cpu" work?

Reply:
This is the icon name provided by elp

Reply:
Alright, let's keep it as-is